content/documentation/interfaces.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Interfaces"
+weight = 13
++++
+
+## Defining interfaces
+
+An interface in Phel defines an abstract set of functions. It is directly mapped to a PHP interface. An interface can be defined by using the `definterface` macro:
+
+```phel
+(definterface MyInterface
+  (foo [this] "foo doc string")
+  (bar [this a b] "bar doc string"))
+```
+
+Each method in the interface has at least one parameter that is bound to the current object (`$this`) later in the implementation. An optional documenation string can be provide a last argument of each method. Additionally the macro defines functions for each method of the interface.
```

Comment:
```suggestion
Each method in the interface has at least one parameter that is bound to the current object (`$this`) later in the implementation. An optional documentation string can be provide as last argument of each method. Additionally the macro defines functions for each method of the interface.
```

Reply:
Is the suggestion `An optional documentation string can be provide as last argument of each method.` correct?